src/System Application/App/Azure AD Tenant/src/AzureADTenant.Codeunit.al

```diff
@@ -37,5 +37,29 @@ codeunit 433 "Azure AD Tenant"
     begin
         exit(AzureADTenantImpl.GetAadTenantDomainName());
     end;
+
+    /// <summary>
+    /// Gets the current Microsoft Entra tenant registered country letter code.
+    /// Visit Microsoft Admin Cententer to view or edit Organizational Information.
+    /// If the Microsoft Graph API cannot be reached, the error is displayed.
+    /// </summary>
+    /// <returns>Country letter code</returns>
+    /// <error>Cannot retrieve the Microsoft Entra tenant country letter code.</error>
+    procedure GetCountryLetterCode(): Text;
+    begin
+        exit(AzureADTenantImpl.GetCountryLetterCode());
+    end;
+
+    /// <summary>
+    /// Gets the current Microsoft Entra tenant registered preferred language.
+    /// Visit Microsoft Admin Cententer to view or edit Organizational Information.
+    /// If the Microsoft Graph API cannot be reached, the error is displayed.
+    /// </summary>
+    /// <returns>Preferred Language</returns>
+    /// <error>Cannot retrieve the Microsoft Entra tenant preferred language.</error>
+    procedure GetPreferredLanguage(): Text;
```

Comment:
I would be nice if this specified what format it returns: lcid? iso code? full name? in local language or in english?

It's also likely that whatever this returns would have to get converted to something known in the Language table/module. Depending on how this returns we may need a function there to handle that bit.

Reply:
Fair point, I'll rip the docs from graph.